docs/book/src/reference/metrics.md

```diff
@@ -3,6 +3,38 @@
 By default, controller-runtime builds a global prometheus registry and
 publishes a collection of performance metrics for each controller.
 
+## Exporting Metrics for Prometheus
+
+To export the metrics we will use the Prometheus Operator. Following the steps to do this setup.
+
+1. Install Prometheus and Prometheus Operator.
+We recommend using [kube-prometheus](https://github.com/coreos/kube-prometheus#installing) in production if you don't have your own monitoring system.
+If you are just experimenting, you can only install Prometheus and Prometheus Operator.
+2. Then, enable the project to export the metrics by uncommenting the line `- ../prometheus` in the `config/default/kustomization.yaml` to allow 
+it creates the `ServiceMonitor` resource which is used to do this export. Following an example.
+
+```yaml
+# [PROMETHEUS] To enable prometheus monitor, uncomment all sections with 'PROMETHEUS'.
+- ../prometheus
+```
+
+Note, that now when you install your project in the cluster it will create the
+`ServiceMonitor` to export the metrics. To check the ServiceMonitor, 
+run `kubectl get ServiceMonitor -n <project>-system`. Following an example.
```

Comment:
```suggestion
run `kubectl get ServiceMonitor -n <project>-system`.
```

Reply:
We need to make clear that it is just an example. I changed it by `See an example:`